src/client/datascience/ipywidgets/ipyWidgetMessageDispatcher.ts

```diff
@@ -395,6 +395,7 @@ export class IPyWidgetMessageDispatcher implements IIPyWidgetMessageDispatcher {
             this.notebook = await this.notebookProvider.getOrCreateNotebook({
                 identity: this.notebookIdentity,
                 resource: this.notebookIdentity,
+                disableUI: true,
                 getOnly: true
```

Comment:
The API is not great, if its `getOnly`, then the value of `disableUI` should not matter at all.
Ideally we need two seprate methods, one to `get` (this will get if found, else return undefined) and the other `getOrCreate` (this will get if found, else create)
This is the pattern we follow elsewhere.

Makes the `getOnly` argument unnecessary and avoids the need for `disableUI` in `getOnly` scenarios.